src/helpers/generate/birthdayList.ts

```diff
@@ -10,27 +10,29 @@ import { ARROW_RIGHT, IMG_CAKE } from '../provide/environment';
 import { formatDateForDisplay, numberToMonthName } from '../utils/date';
 import { envParseNumber } from '@skyra/env-utilities';
 import { generateDefaultEmbed } from '../../lib/utils/embed';
+import dayjs from 'dayjs';
 
 export async function generateBirthdayList(page_id: number, guild_id: string) {
-	const allBirthdaysByGuild = await container.utilities.birthday.get.BirthdaysByGuildId(guild_id);
-	if (!isNullOrUndefinedOrEmpty(allBirthdaysByGuild)) {
-		// sort all birthdays by day and month
-		const sortedBirthdays = sortByDayAndMonth(allBirthdaysByGuild);
-		// split the sorted birthdays into multiple lists
-		const splitBirthdayList = getBirthdaysAsLists(sortedBirthdays, envParseNumber('MAX_BIRTHDAYS_PER_SITE', 80));
-		// get the birthdays for the current page
-		const birthdays = splitBirthdayList.birthdays[getIndexFromPage(page_id)];
-		// TODO: Should only contain the birthdays for the current page (80 birthdays)
-
-		const finalList = prepareBirthdays(birthdays);
-		const embed = await createEmbed(guild_id, finalList);
-
-		const components = generateComponents(page_id, splitBirthdayList.listAmount);
-		return { embed, components };
-	}
-	container.logger.info('no birthdays');
-	const embed = await createEmbed(guild_id, []);
-	return { embed, components: [] };
+	const guild = await container.prisma.guild.findUnique({

```

Comment:
is there a possibility, to only retrieve not disabled birthdays instead of filtering them out later?